src/components/NavBar.tsx

```diff
@@ -17,10 +18,11 @@ const NavBar = () => {
         </Link>
       </div>
       <div className="font-playfair text-naama-blue-200 flex gap-x-12 rounded-full bg-white px-10 py-2 text-lg opacity-75">
-        <Link href="/about">About</Link>
-        <Link href="/calendar">Calendar</Link>
-        <Link href="/board">Board</Link>
-        <Link href="/gallery">Gallery</Link>
+        {Links.map((Links) => (
+          <Link key={Links.label} href={Links.href}>
+            {Links.label}
+          </Link>
```

Comment:
```suggestion
        {Links.map(({label, href}) => (
          <Link key={label} href={href}>
            {label}
          </Link>
```

use destructuring for more readable code